sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/VirtualMachineScaleSet.cs

```diff
@@ -78,15 +80,19 @@ public VirtualMachineScaleSet()
         /// Virtual Machine Scale Set. For instance: whether the Virtual
         /// Machines have the capability to support attaching managed data
         /// disks with UltraSSD_LRS storage account type.</param>
+        /// <param name="scaleInPolicy">Specifies the scale-in policy that
```

Comment:
This generated property is not part of my swagger change.

Reply:
It was added by another previous Swagger PR.  You don't have to remove it.  Just leave it.